Jenkinsfile

```diff
@@ -48,129 +44,103 @@ pipeline {
+                                bat 'mvn install -B -pl !%DOCS% -DskipStatic=true -DskipTests=true -T 1C'
+                                bat 'mvn clean install -B -T 1C -pl !%ITESTS% -Dgib.enabled=true -Dgib.referenceBranch=/refs/remotes/origin/%CHANGE_TARGET%'
+                                bat 'mvn install -B -pl %ITESTS% -nsu'
                             }
                         }
                     }
-                )
+                }
             }
         }
         // The full build will be run against all regular branches
         stage('Full Build') {
             when { expression { env.CHANGE_ID == null } }
-            // TODO DDF-2971 refactor this stage from scripted syntax to declarative syntax to match the rest of the stages - https://issues.jenkins-ci.org/browse/JENKINS-41334
-            steps{
-                parallel(
-                    linux: {
-                        node('linux-large') {
-                            retry(3) {
-                                checkout scm
-                            }
-                            timeout(time: 3, unit: 'HOURS') {
-                                // TODO: Maven downgraded to work around a linux build issue. Falling back to system java to work around a linux build issue. re-investigate upgrading later
-                                withMaven(maven: 'Maven 3.3.9', globalMavenSettingsConfig: 'default-global-settings', mavenSettingsConfig: 'codice-maven-settings', mavenOpts: '${LARGE_MVN_OPTS} ${LINUX_MVN_RANDOM}') {
-                                    sh 'mvn clean install -B -T 1C -pl !$ITESTS'
-                                    sh 'mvn install -B -pl $ITESTS -nsu'
-                                }
+            parallel {
+                stage ('Linux') {
+                    steps {
+                        timeout(time: 3, unit: 'HOURS') {
+                            // TODO: Maven downgraded to work around a linux build issue. Falling back to system java to work around a linux build issue. re-investigate upgrading later
+                            withMaven(maven: 'Maven 3.3.9', globalMavenSettingsConfig: 'default-global-settings', mavenSettingsConfig: 'codice-maven-settings', mavenOpts: '${LARGE_MVN_OPTS} ${LINUX_MVN_RANDOM}') {
+                                sh 'mvn clean install -B -T 1C -pl !$ITESTS'
+                                sh 'mvn install -B -pl $ITESTS -nsu'
                             }
                         }
-                    },
-                    windows: {
-                        node('proxmox-windows') {
-                            bat 'git config --system core.longpaths true'
```

Comment:
@mcalcote any way we can still advise the longpath support? I know that `server-2016-large` supports it by default, but under the current configuration we are still serving that node label using Server 2012 until @oconnormi is done with the Server 2016 Jenkins slaves.

Reply:
@LinkMJB it's actually now pointing at the real server-2016 nodes 😄